src/crush/CrushWrapper.cc

```diff
@@ -1905,7 +1905,7 @@ int CrushWrapper::update_device_class(int id,
   }
 
   if (class_map.count(id) != 0 && class_map[id] == class_id) {
-    *ss << name << " already set to class " << class_name;
+    *ss << name << " already set to class " << class_name << ". ";
```

Comment:
i'd suggest add the separator in `OSDMonitor::prepare_command_impl()` instead. as there is chance that we put more output after this message,  if the osd's is already set to expected device class.